judgments/views.py

```diff
@@ -107,7 +107,11 @@ def advanced_search(request):
         context["paginator"] = paginator(int(page), model.total)
 
         context["query_string"] = "&".join(
-            [f'{key}={query_params[key] or ""}' for key in query_params]
+            [
+                f'{key}={query_params[key] or ""}'
+                for key in query_params
+                if key not in "order"
```

Comment:
I'm not sure what this line is doing - is it just to eliminate the `order` param from the query_string?

Reply:
Yeah my thought was to remove it from the params so it could be overwritten, but I'm not sure if this is the best way!

Reply:
I feel like this is something that should be doable with a bit of standard code, like `URI.encode_www_form(hash)` in Ruby. Then you could just overwrite the order in the hash and convert. But I don't know the python equivalent.

Reply:
Ah, this is it: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode